contrib/netbox.env

```diff
@@ -0,0 +1,15 @@
+# Name is the Process Name
+#
+Name = 'Netbox'
+
+# ConfigPath is the path to the gunicorn config file.
+#
+ConfigPath=/usr/local/netbox/gunicorn.conf
```

Comment:
Shows `/usr/local/netbox` when rest of documentation is using `/opt/netbox`

Reply:
Should be pushing a new version shortly with this change.  It was an oversight.